routes.go

```diff
@@ -97,14 +97,23 @@ func upload(c *gin.Context) {
 	AddLog(1, "uid: "+plug.Owner+"uploaded plug s3id"+plug.S3ID)
 	c.Data(http.StatusOK, "text/html", []byte(`
 	<html>
-	<body>
-		<h2>Uploaded a Plug!</h2>
-		<p>Take a look at what you uploaded! (This does not count towards the views for your Plug!)</p>
-		<p>Due to High Volumes your Plug MUST be approved by a member of any of the following groups (drink, eboard, rtp)! Check /admin</p>
-		<div>
-			<img src="`+S3PresignPlug(plug).String()+`"></img>
-		</div>
-	</body>
+		<body style="background-image:url('https://i.imgur.com/YWfFxsh.png');" link="white" vlink="white" alink="white">
```

Comment:
You could move this over to a template, and just have the `S3PresignPlug(plug).String()` bit passed in as a variable to the template.